### PR TITLE
Adjust log lines

### PR DESCRIPTION
**Pull request by eliranek1**
After looking at telemetry, there was some overlogging that was happening.

Also, cleaned up some of the log lines to prevent confusion

### CODE REVIEW

**Review by jell0wed — APPROVED** [review]

**Comment by eliranek1** [comment]
FYI for reviewers - look at the first commit for the actual changes. The second commit was me running prettier

**Review by mlaw- — APPROVED** [review]

**Comment by thomastay** [comment]
@eliranek1 i think your prettier version is newer than the ObjectStoreProvider one, it's causing the PR build to fail. Can you run `yarn && yarn prettier` from the package root?

File: src/IndexedDbProvider.ts [hunk 1]
```diff
@@ -222,7 +222,9 @@ export class IndexedDbProvider extends DbProvider {
       }
 
       // Create all stores
-      this.logWriter.log(`Creating stores as part of upgrade process`);
+      if (isActualUpgrade) {
+        this.logWriter.log(`Creating stores as part of upgrade process`);
```
Comment (M1Les) [inline src/IndexedDbProvider.ts:226]: It's a bit confusing, but actual difference between isActualUpgrade and isActualMigration is that isActualUpgrade is a db wipe -- last usable version bump vs schema version bump. Last usable version update means that we wipe the database and them apply migrations. Only version bump means that we do an in-place upgrade.

So this should be more accurate: Creating stores after db wipe due to lastUsableVersion change.

File: src/IndexedDbProvider.ts [hunk 2]
```diff
@@ -428,12 +426,18 @@ export class IndexedDbProvider extends DbProvider {
 
     return promise.then(
       (db) => {
-        this.logWriter.log(
-          `Waiting for migrationPutters: ${migrationPutters.length} to complete for db`,
-          { dbName }
-        );
+        // OnUpgradeNeeded will get called for new DBs too so we want to avoid logging those
```
Comment (M1Les) [inline src/IndexedDbProvider.ts:429]: Can you please add a log on line 375: if needsMigrate: "schema changes require rebuilding indices" and if false "schema changes did not require rebuilding indices"

File: src/IndexedDbProvider.ts [hunk 3]
```diff
@@ -428,12 +426,18 @@ export class IndexedDbProvider extends DbProvider {
...
+        const isActualMigration = migrationPutters.length > 0;
+        if (isActualMigration) {
```
Comment (M1Les) [inline src/IndexedDbProvider.ts:431]: technically, we can just check needsMigrate -- without it, migrationPutters will be empty.

**Review by M1Les — CHANGES_REQUESTED** [review]

File: src/IndexedDbProvider.ts (hunk 2 above)
Comment (danielmsft) [reply src/IndexedDbProvider.ts:437]: updated, now using this pull request: https://github.com/microsoft/ObjectStoreProvider/pull/56

File: src/IndexedDbProvider.ts (hunk 3 above)
Comment (danielmsft) [reply src/IndexedDbProvider.ts:439]: needsMigrate is out of scope here and don't want to complicate the update, also the logging here makes sense to me. Leaving as is unless you feel strongly.

now using this pull request: https://github.com/microsoft/ObjectStoreProvider/pull/56 for this

**Pull request merged**